search/includes/classes/concurrency-limiter/class-semaphore-backend.php

```diff
@@ -62,4 +62,8 @@ public function dec_value(): void {
 			}
 		}
 	}
+
+	public function get_value(): int {
+		return (int) $this->increments;
```

Comment:
The Bad Thing™ is that we cannot retrieve the value of the semaphore.

`increments` is how many times we acquired the semaphore in the current session (so that we release it as many times as we have acquired it). Ideally, `increments` shouldn't be more than one, but it it still possible to have a larger value if we call ES in, say, `ep_remote_request` hook (we do that in tests).